topics/metabolomics/tutorials/nmr-preprocessing/tutorial.md

```diff
@@ -0,0 +1,1023 @@
+---
+layout: tutorial_hands_on
+
+title: Nuclear Magnetic Resonance: data preprocessing
+zenodo_link: ''
+questions:
+- What are the main steps of untargeted 1H-NMR data preprocessing for metabolomic analyses?
+- How to conduct 1H-NMR-based metabolomic data preprocessing using Galaxy?
+objectives:
+- To understand the steps necessary to perform preprocessing of untargeted 1H-NMR-based metabolomic data.
+- To get familiar with the use of PEPS-NMR-based Galaxy modules dedicated to 1H-NMR data preprocessing.
+- To evaluate the potential of a workflow approach when dealing with 1H-NMR data preprocessing.
+time_estimation: 2H
+key_points:
+- To process untargeted 1H-NMR based metabolomic data preprocessing, various steps are needed.	
+- Resources are available in Galaxy, but do not forget that you need appropriate knowledge to perform a relevant preprocessing.
+contributors:
+- mtremblayfr
+- Cécile Canlet
+- ManonMartin

```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.